addons/Hierarchical_Table_Of_Contents/test/ModelValidationTests.js

```diff
@@ -0,0 +1,45 @@
+TestCase("[Hierarchical Table Of Contents] Model validation", {
+    setUp: function () {
+        this.presenter = AddonHierarchical_Table_Of_Contents_create();
+
+        this.model = {
+            ID: "Hierarchical_Table_Of_Contents1",
+            isValid: "True",
+            IsVisible: "True",
+            displayOnlyChapters: "false",
+            expandDepth: "",
+            height: "275",
+            width: "300",
+            titleLabel: "Table of Contents",
+            langTag: "",
+            showPages: "",
+        };
+    },
+
+    'test depth of expand empty': function () {
```

Comment:
Nie wiem czy to do końca wymóg, ale w testach staramy się stosować konwencję Given-When-Then [LINK](https://martinfowler.com/bliki/GivenWhenThen.html).
W tym przypadku byłoby "test given model with depth of expand empty when validating then sets depth of expand to 0"